docs/content/documentation/ssl.md

```diff
@@ -69,11 +69,16 @@ If `sslmode=verify-full` , the server host name will be verified to make sure it
 The SSL connection will fail if the server certificate cannot be verified. `verify-full` is recommended in most 
 security-sensitive environments.
 
+
+There is a property `sslNegotiation`. This defaults to `postgres` but if set to `direct` it enables a second alternate way to initiate 
```

Comment:
We have `sslmode`, `sslpassword`, `sslkey`, `sslrootcert`. WDYT of having `sslnegotiation` here? It would align with libpq at the same time. By the way, I've asked what they think of `tlsnegotiation` since it is TLS in the first place.

I would prefer `postgresql` rather than `postgres` as both ALPN and the project name are postgresql rather than posgres.

See https://www.postgresql.org/message-id/CAB%3DJe-G2PK94s_N7Pd6zRO8pYZ1NAnvJg%3DL2XQgKg9-eTLSvsA%40mail.gmail.com

Reply:
so you want to move it to use.md ?

Reply:
Well they aren't going to change to `postgresql`  so ...